src/app/screens/AddOwner/layout.js

```diff
@@ -1,15 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { t } from 'i18next';
 import { Field, reduxForm } from 'redux-form';
 
 import InputLabelNew from '~components/InputLabelNew';
 import SimpleSpinner from '~components/SimpleSpinner';
 import AlertInfo from '~components/AlertInfo';
+import Select from 'app/components/Select';
+import { searchRepositories } from 'services/RepositoryService';
+import { parseArrayToObjectOfSelect } from 'utils/array';
+
+function AddOwner({ handleSubmit, onSubmit, reset, error, loading, submitSucceeded, submitFailed }) {
+  const [repository, setRepository] = useState(null);
+
+  const limit = 100;
+  const loadPageOptions = async (q, prevOptions, { page }) => {
+    const {
+      data: { data }
+    } = await searchRepositories(page, q);
+
+    const options = data.length ? parseArrayToObjectOfSelect(data.map(e => e.split('/')[1])) : [];
```

Comment:
e?

Reply:
Sorry! I made this code in two places at some moment, that's why the abstraction. But It wasn't like that and I forgot to remove it.